lib/stripe/resources/charge.rb

```diff
@@ -10,75 +10,9 @@ class Charge < APIResource
 
     custom_method :capture, http_verb: :post
 
-    def refund(params = {}, opts = {})
```

Comment:
Flagged in Slack but to be safe flagging here too: We have code internally that uses `stripe-ruby` and seems to call `.refund` so that code will need to be updated!